src/Pizzeria.Store.Domain/Pizzeria.Store.Domain.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
```

Comment:
This PropertyGroup is not needed as it's globally defined in Directory.Build.props